frontend/javascripts/test/libs/nml.spec.ts

```diff
@@ -230,6 +231,7 @@ test("NML serializing and parsing should yield the same state", async (t) => {
     initialState.tracing,
     enforceSkeletonTracing(initialState.tracing),
     BUILD_INFO,
+    false,
```

Comment:
Do you mean a spec that uses actual transforms or simply with identity transforms as some sort of smoke test?

Reply:
I was thinking of a case where the skeleton is transformed to test that the state is equal after exporting and importing again. This will be influenced by our internal discussion about the NML format I suppose.

Reply:
As per our discussion, I don't think that such a test would add much value right now, because no automatic retransformation should happen upon import.